src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json

```diff
@@ -47,23 +47,23 @@
     },
     "Microsoft.Android.Ref.31": {
       "kind": "framework",
-      "version": "@WORKLOAD_VERSION@"
+      "version": "31.0.101-preview.11.117"
     },
     "Microsoft.Android.Runtime.31.android-arm": {
       "kind": "framework",
-      "version": "@WORKLOAD_VERSION@"
+      "version": "31.0.101-preview.11.117"
     },
     "Microsoft.Android.Runtime.31.android-arm64": {
       "kind": "framework",
-      "version": "@WORKLOAD_VERSION@"
+      "version": "31.0.101-preview.11.117"
     },
     "Microsoft.Android.Runtime.31.android-x86": {
       "kind": "framework",
-      "version": "@WORKLOAD_VERSION@"
+      "version": "31.0.101-preview.11.117"
     },
     "Microsoft.Android.Runtime.31.android-x64": {
       "kind": "framework",
-      "version": "@WORKLOAD_VERSION@"
+      "version": "31.0.101-preview.11.117"
```

Comment:
It seems the choice to keep API 31 as the .NET default is introducing some new complexity... It looks like we're locking the API 31 ref/runtime pack versions to the latest version available on nuget.org. I think this makes sense as we're now only building the net6.0 version of Mono.Android.dll for API 32.

On the .NET MSBuild testing side we may have a new gap in coverage though. I think the default test project will continue to target `net6.0-android31`, which will be running against ref/runtime packs that have already been published and not the ones produced by the build. The positive side of this is that we will ensure that any changes to the SDK continue to work with our previously produced API 31 packs. A potential downside is that we may now miss an issue caused by a change in Mono.Android.csproj. I'm not sure what the best solution for this is at the moment and I don't think it necessarily needs to be addressed in this PR, but it's worth further discussion.

Reply:
Yes, if we end up with 32 as the default in .NET 6, we wouldn't have to put the 31 packs in the workload.

This logic here, allows them to come from NuGet *or* the workload:

https://github.com/xamarin/xamarin-android/blob/5f66c36fc10ae2f159885fafe15b16b871378bd1/build-tools/create-packs/Microsoft.Android.Sdk.proj#L122-L142

I think we should wait and see what we end up doing.